### ISSUE TITLE
Add e1000 network driver support for windows devices

### ISSUE THREAD

**Issue by ayunami2000** [copy/v86#473]
I think this will fix the network issue on newer windows versions (vista, windows 7, etc.). This is supported in qemu as is and in many windows versions as well.

**Comment by copy** [comment]
Would be useful for QNX (https://github.com/copy/v86/issues/412). For Windows it would probably be better to fix bugs in the already existing ne2k code.

**Comment by BenNottelling** [comment]
I thought the most recent version of windows that supported ne2k was longhorn (vista beta)?
Vista you should be able to install the win 2k driver, but I'm not sure a bout anything after that

**Comment by copy** [comment]
> I thought the most recent version of windows that supported ne2k was longhorn (vista beta)?
> Vista you should be able to install the win 2k driver, but I'm not sure a bout anything after that

Indeed, I wasn't aware that. In any case, supporting non-abandonware is not a priority, but a PR would be welcome.

**Comment by PokeyManatee4** [comment]
eh may work it may not never know
